modules/internal/Bytes.chpl

```diff
@@ -431,6 +431,16 @@ module Bytes {
       return createBytesWithOwnedBuffer(buf, length=1, size=size);
     }
 
+    /*
+      :returns: The value of a single-byte bytes as an integer.
```

Comment:
``bytes``

Reply:
I don't understand what this refers to. It used to say "single-byte string" in an earlier commit which is fixed already.

Reply:
I want you to put the word "bytes" in double-backticks like
```
       :returns: The value of a single-byte ``bytes`` as an integer. 
```